docs/kr.tree

```diff
@@ -404,24 +389,53 @@
 			<include from="api-guidelines.tree" origin="api-guidelines" element-id="api-guidelines"/>
 		</toc-element>
 	</toc-element>
-	<toc-element toc-title="Early access preview (EAP)">
-		<toc-element toc-title="Participate" topic="eap.md"/>
-		<toc-element topic="configure-build-for-eap.md" accepts-web-file-names="install-eap-plugin.html"/>
-	</toc-element>
 	<toc-element toc-title="Other resources">
 		<toc-element topic="faq.md"/>
-		<toc-element toc-title="Kotlin compatibility guides">
-			<toc-element topic="compatibility-guide-21.md"/>
-			<toc-element topic="compatibility-guide-20.md"/>
-			<toc-element topic="compatibility-guide-19.md"/>
-			<toc-element topic="compatibility-guide-18.md"/>
-			<toc-element topic="compatibility-guide-1720.md"/>
-			<toc-element topic="compatibility-guide-17.md"/>
-			<toc-element topic="compatibility-guide-16.md"/>
-			<toc-element topic="compatibility-guide-15.md"/>
-			<toc-element topic="compatibility-guide-14.md"/>
-			<toc-element topic="compatibility-guide-13.md"/>
-			<toc-element topic="compatibility-modes.md"/>
+		<toc-element toc-title="Earlier Kotlin versions">
+			<toc-element toc-title="Kotlin 2.0.x">
+				<toc-element toc-title="Kotlin 2.0.20" topic="whatsnew2020.md"/>
+				<toc-element toc-title="Kotlin 2.0.0" topic="whatsnew20.md"/>
+				<toc-element topic="compatibility-guide-20.md"/>
```

Comment:
I think for the left-side menu, the short name "Compatibility guide" would fit better, so the structure of the release is:

**Kotlin 2.0.x**
* Kotlin 2.0.20
* Kotlin 2.0.0
* Compatibility guide

But the page itself would have the full name "Compatibility guide for Kotlin 2.0.x"

Reply:
Do you think we should update the name for the freshest version as well?
I mean that one on line 29.

Reply:
Yeah, definitely, for consistency